src/main/java/org/apache/mesos/state/AbstractStateStore.java

```diff
@@ -0,0 +1,42 @@
+package org.apache.mesos.state;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+
+/**
+ * Abstract StateStore implementation.
+ */
+public abstract class AbstractStateStore implements StateStore {
```

Comment:
These feel like they could be static utility methods, rather than putting them in statestore proper